presets/inference/text-generation/inference_api.py

```diff
@@ -98,7 +117,10 @@ def __post_init__(self): # validate parameters
 combination_type = model_args.pop('combination_type')
 
 app = FastAPI()
+resovled_chat_template = load_chat_template(model_args.pop('chat_template'))
```

Comment:
so just to make sure, now the filename to the chat template is passed in as a default param, we will need to update this on the controller side in future PR?